src/Module.cs

```diff
@@ -316,6 +316,44 @@ public static Module DeserializeFile(Engine engine, string name, string path)
             return new Module(handle, name);
         }
 
+        /// <summary>
+        /// Convert WAT (Web Assembly Text) into WASM bytes
+        /// </summary>
+        /// <param name="wat">A string containing WAT (Web Assembly Text)</param>
+        /// <returns></returns>
```

Comment:
```suggestion
        /// <returns>Returns a byte array containing the WebAssembly module represented by the given text.</returns>
```

Reply:
Oops, I should have filled that one in myself. Sorry about that 🤦‍♂️